WcaOnRails/app/views/delegate_reports/edit.html.erb

```diff
@@ -0,0 +1,20 @@
+<% provide(:title, "Report for #{@competition.name}") %>
+
+<%= render layout: 'nav' do %>
+  <h2>Editing Delegate Report for <%= @competition.name%></h2>
+  <%= simple_form_for(@delegate_report, url: competition_report_path(@competition.id)) do |f| %>
+    <%= f.hidden_field :updated_at %>
```

Comment:
I think at that time there was no `posted_at`, and the `updated_at` was a way to know the posting date.